FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/FirebaseAuthSwiftUI/FUIAuth.swift

```diff
@@ -49,7 +58,7 @@ public class FUIAuth: ObservableObject {
   }
 
   public func getEmailProvider() -> EmailAuthProvider? {
-    return try! providerWithId(providerId: "email") as! EmailAuthProvider
+    return try! providerWithId(providerId: "password") as! EmailAuthProvider
```

Comment:
Please avoid using force unwrap.

Would also recommend using constants (enum, probably) for the provider names.